src/rezplugins/shell/cmd.py

```diff
@@ -352,6 +355,16 @@ def join(cls, command):
     def line_terminator(cls):
         return "\r\n"
 
+    def _alias_map(self, key, value):
+        if self._doskey_alias is None:
+            self._doskey_alias = dict()
+        self._doskey_alias[key] = value
+
+    def _reveal_alias(self, command):
```

Comment:
This could be a little more robust I think. The problem's gonna be if args are passed to the alias, the alias expansion won't happen in that case. Even just taking the first word in `command` and expanding just that, is probably going to work in 99% of cases.

I would also include in a docstring somewhere, an explanation as to why this is here in the first place. Without knowing about the doskey issue it's gonna be difficult for someone coming in fresh to know why this alias expansion is being done.

Reply:
Thanks for review :D

> The problem's gonna be if args are passed to the alias, the alias expansion won't happen in that case.

Yes, maybe `str.startswith` would do the trick. Will test it later.

> I would also include in a docstring somewhere

🙆🏼‍♂️ 👌🏼